plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/VmwareServerDiscoverer.java

```diff
@@ -151,16 +151,8 @@ public VmwareServerDiscoverer() {
                 clusterDetails.put("password", password);
                 _clusterDetailsDao.persist(clusterId, clusterDetails);
             }
-            String updatedInventoryPath = validateCluster(url, vmwareDc);
-            try {
-                if (!URLDecoder.decode(url.getPath(), "UTF-8").equals(updatedInventoryPath)) {
-                    // If url from API doesn't specify DC then update url in database with DC associated with this zone.
-                    clusterDetails.put("url", url.getScheme() + "://" + url.getHost() + updatedInventoryPath);
-                    _clusterDetailsDao.persist(clusterId, clusterDetails);
```

Comment:
@sureshanaparti could not persisting the url in the cluster details cause regression?